### ISSUE TITLE
Add a file extension to debug output

### ISSUE THREAD

**Issue by gravitystorm** [djezzzl/database_consistency#218]
If one of the checks fails with an error, you get a message like:

```
Hey, some checks fail with an error, please open an issue on github at https://github.com/djezzzl/database_consistency.
Attach the created file: database_consistency_2023_10_25_15_46_43
```

Unfortunately github issue comments doesn't allow attaching files without an extension, so the file needs to be renamed first (e.g. to add a `.txt` extension). It would be great if database_consistency generated that debug file with a suitable extension by default.

**Comment by simplexbruno** [comment]
I have the same issue. 
Loaded configurations: .database_consistency.yml, .database_consistency.todo.yml
Hey, some checks fail with an error, please open an issue on github at https://github.com/djezzzl/database_consistency.
Attach the created file: database_consistency_20_11_07_17_52_28

**Comment by djezzzl** [comment]
Hello!

Thank you for reporting this. I have released 1.7.22 with the fix. Hav

**Comment by simplexbruno** [comment]
Thanks! @djezzzl
